indexer/indexing.py

```diff
@@ -92,66 +97,98 @@ async def create_pg_vector_index_if_not_exists(self):
             port=self.db_port,
         )
         try:
-            # Create an asynchronous cursor object to interact with the database
             async with connection.transaction():
-                # Execute the alter table query
                 await connection.execute(
                     "ALTER TABLE langchain_pg_embedding ALTER COLUMN embedding TYPE vector(1536)"
                 )
                 await connection.execute(
                     "CREATE INDEX IF NOT EXISTS langchain_embeddings_hnsw ON langchain_pg_embedding USING hnsw (embedding vector_cosine_ops)"
                 )
         finally:
-            # Close the connection
             await connection.close()
 
     async def index(self, indexer_input: IndexerInput):
+        source_files = []
         source_chunks = []
         counter = 0
         for file in indexer_input.files:
             file_path = os.path.join(os.environ["DOCUMENT_LOCAL_STORAGE_PATH"], file)
```

Comment:
Please use Storage helpers that we have written. 

@shreypandey  can you paste a snippet of what is required here?

Reply:
Sure, will look into it from examples.